website/src/documentation/runtime/environments.md

```diff
@@ -99,6 +99,7 @@ python -m apache_beam.examples.wordcount \
 --environment_config=path/to/container/image
 ```
 
+(Customized containers are not supported with Dataflow runner yet.)
```

Comment:
If custom containers are not supported yet, do the instructions below this work?

Reply:
I think it's still better to include it, it is telling users Dataflow will support custom containers in future, though they are not supported at the moment.

Reply:
Can we at least make that more clear? It's confusing to document features we don't support yet.

Reply:
I removed dataflow part. Can you please take a look?
(Sorry for long due...)